programs/server/config.xml

```diff
@@ -742,9 +742,9 @@
              It also enables 'changeable_in_readonly' constraint type -->
         <settings_constraints_replace_previous>true</settings_constraints_replace_previous>
 
-        <!-- By default, for backward compatibility create table with a specific table engine ignores grant,
+        <!-- By default, for backward compatibility creating table with a specific table engine ignores grant,
              however you can change this behaviour by setting this to true -->
-        <table_engines_require_grant>true</table_engines_require_grant>
+        <table_engines_require_grant>false</table_engines_require_grant>
```

Comment:
It seems I missed that while reviewing https://github.com/ClickHouse/ClickHouse/pull/60117, thank you!

Reply:
@jsc0218 The idea was to enable `table_engines_require_grant` by default **only** in tests, not just for everyone who uses the default config. And we have directory `/tests/config/config.d` to make changes in the default config which will affect only tests.